CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Removed
+- Short flags for `--tenant`, `--clear`, `--domain`, `--auth-mode`, and `--output`.
```

Comment:
I think we should list the actual flags removed, someone glancing at this might easily misread this as removed flag - and see the long names and think those were removed.

Reply:
That makes much more sense than what I've done here. I'll go switch that.